1-org/variables.tf

```diff
@@ -67,6 +67,12 @@ variable "data_access_table_expiration_ms" {
   default     = 2592000000
 }
 
+variable "skip_gcloud_download" {
```

Comment:
@amandakarina is there a reason why we are calling out the need for `gcloud alpha module`?